<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameServer</class>
 <widget class="QMainWindow" name="GameServer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1054</width>
    <height>711</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Skull King</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>341</x>
      <y>62</y>
      <width>160</width>
      <height>200</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>160</width>
      <height>200</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>160</width>
      <height>200</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>508</x>
      <y>61</y>
      <width>160</width>
      <height>200</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>160</width>
      <height>200</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>160</width>
      <height>200</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="card_1">
    <property name="geometry">
     <rect>
      <x>188</x>
      <y>323</y>
      <width>90</width>
      <height>120</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>90</width>
      <height>120</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>90</width>
      <height>120</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/Picture/card pushputton.png);</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="card_2">
    <property name="geometry">
     <rect>
      <x>285</x>
      <y>323</y>
      <width>90</width>
      <height>120</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>90</width>
      <height>120</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>90</width>
      <height>120</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/Picture/card pushputton.png);</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="card_3">
    <property name="geometry">
     <rect>
      <x>382</x>
      <y>323</y>
      <width>90</width>
      <height>120</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>90</width>
      <height>120</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>90</width>
      <height>120</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/Picture/card pushputton.png);</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="card_4">
    <property name="geometry">
     <rect>
      <x>479</x>
      <y>323</y>
      <width>90</width>
      <height>120</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>90</width>
      <height>120</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>90</width>
      <height>120</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/Picture/card pushputton.png);</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="card_5">
    <property name="geometry">
     <rect>
      <x>576</x>
      <y>323</y>
      <width>90</width>
      <height>120</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>90</width>
      <height>120</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>90</width>
      <height>120</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/Picture/card pushputton.png);</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="card_6">
    <property name="geometry">
     <rect>
      <x>673</x>
      <y>323</y>
      <width>90</width>
      <height>120</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>90</width>
      <height>120</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>90</width>
      <height>120</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/Picture/card pushputton.png);</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="card_7">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>323</y>
      <width>90</width>
      <height>120</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>90</width>
      <height>120</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>90</width>
      <height>120</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/Picture/card pushputton.png);</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="card_8">
    <property name="geometry">
     <rect>
      <x>188</x>
      <y>450</y>
      <width>90</width>
      <height>120</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>90</width>
      <height>120</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>90</width>
      <height>120</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/Picture/card pushputton.png);</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="card_9">
    <property name="geometry">
     <rect>
      <x>285</x>
      <y>450</y>
      <width>90</width>
      <height>120</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>90</width>
      <height>120</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>90</width>
      <height>120</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/Picture/card pushputton.png);</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="card_10">
    <property name="geometry">
     <rect>
      <x>382</x>
      <y>450</y>
      <width>90</width>
      <height>120</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>90</width>
      <height>120</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>90</width>
      <height>120</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/Picture/card pushputton.png);</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="card_11">
    <property name="geometry">
     <rect>
      <x>479</x>
      <y>450</y>
      <width>90</width>
      <height>120</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>90</width>
      <height>120</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>90</width>
      <height>120</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/Picture/card pushputton.png);</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="card_12">
    <property name="geometry">
     <rect>
      <x>576</x>
      <y>450</y>
      <width>90</width>
      <height>120</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>90</width>
      <height>120</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>90</width>
      <height>120</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/Picture/card pushputton.png);</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="card_13">
    <property name="geometry">
     <rect>
      <x>673</x>
      <y>450</y>
      <width>90</width>
      <height>120</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>90</width>
      <height>120</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>90</width>
      <height>120</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/Picture/card pushputton.png);</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="card_14">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>450</y>
      <width>90</width>
      <height>120</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>90</width>
      <height>120</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>90</width>
      <height>120</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/Picture/card pushputton.png);</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="NumberOfPredict">
    <property name="geometry">
     <rect>
      <x>605</x>
      <y>581</y>
      <width>180</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>180</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>180</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Segoe Script</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lablePredict">
    <property name="geometry">
     <rect>
      <x>206</x>
      <y>581</y>
      <width>391</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>391</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>391</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Segoe Script</family>
     </font>
    </property>
    <property name="text">
     <string>   predict number of sets you are going to win:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Ok">
    <property name="geometry">
     <rect>
      <x>792</x>
      <y>581</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Segoe Script</family>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Ok</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>341</x>
      <y>269</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/Picture/card pushputton.png);
font: 9pt &quot;Segoe Script&quot;;</string>
    </property>
    <property name="text">
     <string>you</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>270</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/Picture/card pushputton.png);
font: 9pt &quot;Segoe Script&quot;;</string>
    </property>
    <property name="text">
     <string>opponent</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1061</width>
      <height>701</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/Picture/backgame.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Loading">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1061</width>
      <height>701</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Ok_2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>530</y>
      <width>90</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>90</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>90</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Segoe Script</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qconicalgradient(cx:0, cy:1, angle:0, stop:0.0673077 rgba(156, 105, 60, 255), stop:1 rgba(255, 255, 255, 255));
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>Ok</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="IpServer">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>530</y>
      <width>300</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Segoe Script</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qconicalgradient(cx:0, cy:1, angle:0, stop:0.0673077 rgba(156, 105, 60, 255), stop:1 rgba(255, 255, 255, 255));
border-radius:10px;
</string>
    </property>
    <property name="placeholderText">
     <string>Enter Server IP address</string>
    </property>
   </widget>
   <widget class="QListWidget" name="YourIp">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>590</y>
      <width>400</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>400</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>400</width>
      <height>40</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qconicalgradient(cx:0, cy:1, angle:0, stop:0.0673077 rgba(156, 105, 60, 255), stop:1 rgba(255, 255, 255, 255));
font: 15pt &quot;Segoe UI&quot;;
font: 9pt &quot;Segoe Script&quot;;
border-radius:10px;
</string>
    </property>
    <item>
     <property name="text">
      <string>Your Ip:</string>
     </property>
     <property name="font">
      <font>
       <family>Segoe Script</family>
       <pointsize>13</pointsize>
      </font>
     </property>
    </item>
   </widget>
   <zorder>background</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>card_1</zorder>
   <zorder>card_2</zorder>
   <zorder>card_3</zorder>
   <zorder>card_4</zorder>
   <zorder>card_5</zorder>
   <zorder>card_6</zorder>
   <zorder>card_7</zorder>
   <zorder>card_8</zorder>
   <zorder>card_9</zorder>
   <zorder>card_10</zorder>
   <zorder>card_11</zorder>
   <zorder>card_12</zorder>
   <zorder>card_13</zorder>
   <zorder>card_14</zorder>
   <zorder>NumberOfPredict</zorder>
   <zorder>lablePredict</zorder>
   <zorder>Ok</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>Loading</zorder>
   <zorder>Ok_2</zorder>
   <zorder>IpServer</zorder>
   <zorder>YourIp</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
